# many IMC maneuvers should fit in this message
# and we should just decide what to do with the data
# depending on the maneuver name or id
# i will take the entity state
imc_ros_interface/Header header
uint8 state #
uint8 flags
string description
